Documentation/Signoffs/benjaminreed-signoff-hdmiswitcher.md

```diff
@@ -0,0 +1,66 @@
+
+*Constraints:*
+
+*HDMI Compatibility:* As HDMI is standard in the modern era, the multiplexer needs to be compatible with such an input. The RRS Acquisition is designed to record video at a resolution of 720p. Furthermore, the Nintendo Switch is rated to display data at a resolution of 1080p while in TV mode, which is the mode that the Nintendo Switch will be in while using the bike [1]. The multiplexer must therefore be rated to manage resolutions up to 1080p. It should be commented on that the use of HDMI allows for ease of accessibility for most of the population, especially those who can purchase the MK bike and the RRS kit.
+
+The BHS can display up to 4K resolutions with the caveat that the HDMI cables are no longer than five meters [2]. This is sufficient for the Nintendo Switch and the RRS.
+
+*HDMI Quantity:* As the switcher is expected to manage three ports, the multiplexer must also be capable of doing so. The switcher must have one output port and be capable of switching between two input ports. The BHS meets this criterion [1].
+
+*BHS Switch Ratings: *The BHS's voltage maximums for the switch cannot be determined until the BHS is purchased, as there is no datasheet for the product. An analysis will be conducted to determine the voltage of the signal sent when the button is pressed. Since the product can be purchased off Amazon with two-day shipping, analysis can begin quickly. Once analysis is completed, a follow-up design will be completed.
+
+*Relay Ratings:* As the Raspberry Pi only supplies 3.3 V and a range of -2 mA to 2 mA outputs, a relay with sufficient ratings for those attributes. Additionally, the ratings for the switch pins of the relay must be able to withstand the voltage and current values that the switcher's button is presently located. A proper relay cannot be fully determined until the BHS is analyzed.
+
+*Video State Reset:* Since the BHS utilizes a single button switch, an error can occur when turning off the Raspberry Pi where the state of the switcher and the expected state according to the Raspberry Pi disagree. It is imperative that the UI has a contingency to ensure that the correct display appears on the user's monitor. This is an issue that will be resolved via a UI program. Specifically, the program will read the switcher's current state so if the MK/RRS system is ever unplugged, the Raspberry Pi knows the current state of the switcher and can determine if switching outputs is necessary. Analysis of the BHS is necessary to ensure that this is possible.
+
+*Buildable Schematic:*
```

Comment:
If this is going to be connected to the raspberry pi, then it needs to be added to that schematic and the updated version put here. You can't simply reference one of the GPIO and expect this to be buildable by a 3rd party.

Reply:
Since this is part of a greater schematic it needs to be consolidated. This is a duplication. Now, if something changes on the other schematic, it won't be reflected here. There should only be one schematic for one device.